README.md

```diff
@@ -16,7 +16,7 @@ After these responsibilities are met, the contract ends and another
 service or consumer takes over or the request terminates in a response from this
 service.
 
-This service works in tandem with the [NYPL Checkin Request Service](https://github.com/NYPL/checkin-request-service).
+This service works in tandem with the [NYPL Checkin Request Service](https://github.com/NYPL/checkin-request-service). The two services now are consumed by [Cancel Request Consumer](https://github.com/NYPL/cancel-request-consumer) for canceling a hold on ReCap. The way to perform the cancelling is to check out and check in the item right away.
```

Comment:
"for canceling a hold on ReCap" should be "for canceling a hold on a ReCap item in Sierra"